DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/AdminFields/AddAdminField.cshtml

```diff
@@ -0,0 +1,92 @@
+﻿@inject IConfiguration configuration
+@using DigitalLearningSolutions.Web.Controllers.TrackingSystem.CourseSetup
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using Microsoft.Extensions.Configuration
+@model AddAdminFieldViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/shared/cardWithButtons.css")" asp-append-version="true">
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = "Add Course Admin Field";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+  var cancelLinkData = new Dictionary<string, string> { { "customisationId", Model.CustomisationId.ToString() } };
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(AddAdminFieldViewModel.CustomPromptId), nameof(AddAdminFieldViewModel.Answer) }" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Add course admin field</h1>
+  </div>
+</div>
+
+<form method="post" novalidate asp-action="AddAdminField">
+  <vc:select-list asp-for="CustomPromptId"
+                  label="Field name"
+                  value="@Model.CustomPromptId.ToString()"
+                  hint-text=""
+                  deselectable="false"
+                  css-class="nhsuk-u-width-one-half"
+                  default-option="Select a prompt name"
+                  select-list-options="@ViewBag.CoursePromptNameOptions" />
+
+  <div class="hidden-submit">
+    <button name="action" class="nhsuk-button" value="@AdminFieldsController.AddPromptAction">Add</button>
+  </div>

```

Comment:
This button needs the changes from HEEDLS-488 adding to it so that it works in IE (see EditAdminField)

Reply:
Ah yes, forgot about that. I've added the aria-hidden and tabindex